libraries/acme.rb

```diff
@@ -25,15 +25,18 @@
 end
 
 def acme_client
-  Chef::Application.fatal!('Acme requires that a contact is specified') if node['acme']['contact'].empty?
```

Comment:
Would be nice to still have this check in and have it fire if both the contact from the attributes as the one from the resource are empty

Reply:
We'd like to allow for certificates without contacts.

I could change this to `warning` level. 

It's more complicated, but I could also change the default to `nil` and fail if that isn't changed but an empty Array would still be considered valid.

Reply:
Checked the RFC and it seems providing a contact is now optional. So with that in mind the behaviour introduced with this change should be fine.